src/Symfony/Component/DomCrawler/composer.json

```diff
@@ -21,7 +21,11 @@
         "symfony/polyfill-mbstring": "~1.0"
     },
     "require-dev": {
-        "symfony/css-selector": "~3.4|~4.0"
+        "symfony/css-selector": "~3.4|~4.0",
+        "masterminds/html5": "^2.6"
+    },
+    "conflict": {
+        "masterminds/html5": "<2.6"
     },
     "suggest": {
         "symfony/css-selector": ""
```

Comment:
Shouldn't there be an entry here, that describes that you can load `masterminds/html5`?

Reply:
indeed, that would make sense.

Reply:
I'm against adding things under `suggest`, nobody reads them anyway. I would even go as far as removing the existing entries :)

Reply:
I always read them!